CoEpi/domain/model/KeyValueStore.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+class KeyValueStore {
+    static let boolDefaults = ["showOnboarding": true]
+    
+    static func putBool(key: String, value: Bool) {
```

Comment:
I'd make an enum for the keys, to make it safer. E.g:

```swift
enum PreferenceKey {
    case seenOnboardidng
}
```
See the [Android implementation](https://github.com/Co-Epi/app-android/blob/develop/app/src/main/java/org/coepi/android/system/Preferences.kt)

(You can ignore `clear` for now)

Reply:
Keys are now an enum.